torchaudio/prototype/__init__.py

```diff
@@ -0,0 +1,11 @@
+from .transducer import (
+    compute_alphas,
+    compute_betas,
+    TransducerLoss,
+)
+
+__all__ = [
+    'compute_alphas',
+    'compute_betas',
```

Comment:
we'll need feedback on whether we want to expose these two in python, please add this to the follows-up needed (in the description)